packages/amplify-util-mock/src/api/api.ts

```diff
@@ -383,12 +386,18 @@ export class APITest {
     if (!name) {
       throw new AmplifyFault('MockProcessFault', {
         message: 'No AppSync API is added to the project',
-        link: AMPLIFY_SUPPORT_DOCS.CLI_GRAPHQL_TROUBLESHOOTING.url,
+        resolution: `Use 'amplify init' in the root of your app directory to create a new environment.`,
+        link: `${errorSuffix}`,
       });
     }
     return name;
   }
 
+// extracted method for testing purposes only
+  public async getAppSyncApiPublic(context) {
+    return await this.getAppSyncAPI(context);
+  }
+
```

Comment:
Why do you need a public wrapper around this private method? Is it for testing?

Reply:
yes Phani, for testing had to extract this method